docs/sphinx/using/cudaq/platform.rst

```diff
@@ -104,32 +116,134 @@ should be used.
 
 An example of MPI distribution mode usage in both C++ and Python is given below:
 
-C++
-^^^
+.. tab:: C++
 
-.. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
-    :language: cpp
-    :start-after: [Begin Documentation]
-    :end-before: [End Documentation]
+    .. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
+        :language: cpp
+        :start-after: [Begin Documentation]
+        :end-before: [End Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    nvq++ file.cpp -target nvidia-mqpu
-    mpirun -np <N> ./a.out
+        nvq++ file.cpp -target nvidia-mqpu
+        mpirun -np <N> a.out
 
 
-Python
-^^^^^^
+.. tab:: Python
 
-.. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
-    :language: python
-    :start-after: [Begin Documentation]
+    .. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
+        :language: python
+        :start-after: [Begin Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    mpirun -np <N> python3 file.py
+        mpirun -np <N> python3 file.py
 
 In the above example, the parallel distribution mode was set to :code:`mpi` using :code:`cudaq::parallel::mpi` in C++ or :code:`cudaq.parallel.mpi` in Python.
 CUDA Quantum provides MPI utility functions to initialize, finalize, or query (rank, size, etc.) the MPI runtime. 
 Last but not least, the compiled executable (C++) or Python script needs to be launched with an appropriate MPI command, 
 e.g., :code:`mpirun`, :code:`mpiexec`, :code:`srun`, etc.
+
+Multi-GPU QPU simulation
```

Comment:
```suggestion
Remote `MQPU` Platform
```
I think it may be clearer to understand what the platform is if we use the name `remote-mqpu` instead, similar to `nvidia-mqpu`.